lib/livebook_web/live/session_live.ex

```diff
@@ -888,6 +932,8 @@ defmodule LivebookWeb.SessionLive do
   end
 
   def handle_event("format_code", %{"code" => code}, socket) do
+    assert_policy!(socket, :edit)
```

Comment:
Oh, I think we don't use this anywhere, as it became a part of the `"intellisense_request"`, so feel free to remove this altogether :)